### PR TITLE
Setup Linters config and add basic html,css files.

### PR DESCRIPTION
**Pull request by khitermedachraf**
I set up the Linters config to detect errors and follow the community rules.
then I add initial HTML structures, CSS rules to build the header and the headline section of my portfolio including menu icons, social media icons, and a button.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style.css" />
+    <title>Achraf's Portfolio</title>
+  </head>
+
+  <body>
+    <header class="flex-div">
+      <h1>Welcome</h1>
```
Comment (aimemalaika) [inline index.html]: - [x] the logo of the website should be in `a` tag not `h1` 
change `<h1>Welcome</h1>` to `<a href="#">Welcome</a>`

**Review by aimemalaika — CHANGES_REQUESTED** [review]
Hi @khitermedachraf, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

the-simpsons-homer-simpson

## Highlights
- properly setup linter ✅ 
- professionally written README ✅ 
- The content is aligned with flexbox ✅ 

## Required Changes ♻️

- [x] Please check the comment for requirements.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (khitermedachraf) [reply index.html]: Ahh Got it, Thank you so much, Sir.

Comment (khitermedachraf) [reply index.html]: Hi @aimemalaika, I am done with the requirement changes Sir.

**Review by BenMukebo — APPROVED** [review]
<h1>Status: Approve  :champagne:  :+1: :confetti_ball: 

### **Hi @khitermedachraf ** , 




### **Congratulations! 🎉 Project Approved ✅**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 


### To Highlight :1st_place_medal: 
- No Linter errors :heavy_check_mark: 
- Your Design looks awesome, keep rocking :rocket: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏💻  

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**